docs/src/components/components/action/button/README.md

```diff
@@ -0,0 +1,22 @@
+
+# Button
+
+The **button** component is used when you want the user to take an
+action. Buttons should be used for changing the state of a page or transforming
+data.
+
+## Use
+
+To use the **button** component, add the classes `usa-button action` to a
+`<button>` element.
+
+In addition, add any of the following classes to the button to style the button
+based on status.
+
+| Class   | Description |
+| ------   | -----------|
+| `action-primary usa-button-primary` | When to use |
+| `action-base usa-button-base`       | When to use |
+| `action-warning usa-button-warning`  | When to use |
+| `action-cautious usa-button-cautious`    | When to use |
+| `action-finish usa-button-finish`    | When to use |
```

Comment:
it looks like the `disabled` prop renders `usa-button-disabled` in the templates in addition to any `style` props passed